.github/workflows/upgrade.yml

```
@@ -0,0 +1,38 @@
+name: Upgrade
+on:
+  workflow_dispatch:
+      inputs:
+        network:
+            description: 'Network'
+            required: true
+            type: choice
+            options:
+            - hardhat
+            - bellecour
+jobs:
+  upgrade:
+    runs-on: ubuntu-latest
+    permissions:
+        contents: write # required by git-auto-commit-action
+    environment: ${{ inputs.network }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Setup Node
```

```suggestion
    - name: Setup NodeJS
```
to specify we're not talking about the runner Node.

---

`Node.js` ?

---

See [Rename step to Node.js](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/64/commits/0c9718e0d470632a83a490cf9247f135aa14f4b7)